paper/paper.md

```diff
@@ -60,6 +60,24 @@ Scientific data across experimental physics and materials science remains largel
 
 # NeXus reader and annotator (read\_nexus) (Sandor)
 
+__read_nexus__ is a command line tool to annotate NeXus data files. The tool is registered by the Python ecosystem in the PATH and so it is practicly made available for immediate command line use in the environment the pynxtools python module is installed. The available command line options are explained if the argument _--help_ is passed when _read\_nexus_ is launched. These options allow selecting a NeXus file and either documenting all or part of its content. Note that with no specific NeXus file provided as an argument, the tool will use by default an example NeXus file shipped together with the package.
+
+_annotations -_ As it is expained above, NeXus defines a large amount of concepts, the so called NeXus Vocabulary (https://manual.nexusformat.org/classes/index.html#classes-vocabulary-downloads). These include _Attributes_ and _Fields_ being organized in _Groups_ and their subgroups. NeXus files store their data items following the hierarchy of these groups (see details in https://manual.nexusformat.org/datarules.html). _read\_nexus_ traverses the input NeXus file and interprets each data item by following its group path and connecting it to the concept of the specific vocabulary item. It logs the concept's definition and, since NeXus concepts follow inheritance, those of its superclass concepts along the inheritance chain. If a data item in a NeXus file does not belong to any concepts (which is allowed by the NeXus standard), _read\_nexus_ logs the message 'NOT IN SCHEMA'.  
```

Comment:
above (better specific reference to paragraph)
instead of "large amount" give "..., NeXus defines .... concepts" give exact number and reference to the commit where you counted the number in.

Reply:
Reformulate "The tool reports the definition of each concept" make a full stop definately before the "and, since", this and, since breaks the reading flow and is a related but not the same idea so ok to have a new sentence